WalletWasabi/Http/Models/StartLine.cs

```diff
@@ -45,11 +45,6 @@ public static List<string> GetParts(string startLineString)
 
 					if (parts.Count == 2)
 					{
-						if (part == null)
```

Comment:
Previously we check if `part` is `null` or `empty` so, it can not be null at this point.